framework/db/mysql/Schema.php

```diff
@@ -174,7 +174,7 @@ protected function loadColumnSchema($info)
         $column->phpType = $this->getColumnPhpType($column);
 
         if (!$column->isPrimaryKey) {
-            if ($column->type === 'timestamp' && $info['default'] === 'CURRENT_TIMESTAMP') {
+            if (($column->type === 'timestamp' || $column->type ==='datetime') && $info['default'] === 'CURRENT_TIMESTAMP') {
```

Comment:
It is supported since MySQL 5.6

https://dev.mysql.com/doc/refman/5.6/en/timestamp-initialization.html#idm139764692637360